src/main/java/com/example/module2/StudentMain.java

```diff
@@ -0,0 +1,124 @@
+package com.example.module2;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class StudentMain {
+    public static void main(String[] args) throws ParseException {
+        SimpleDateFormat format = new SimpleDateFormat("dd.MM.yyyy");
```

Comment:
Please move format onto StudentMain field.
Like this:
```
public class StudentMain {
    private final static SimpleDateFormat format = new SimpleDateFormat("dd.MM.yyyy");
    ...
}
```
After this you will be able to use this format in multiple methods in this class.